### DISCUSSION TITLE
no worker in ploting status

### DISCUSSION THREAD

**Discussion by robertbigk** [guydavis/machinaris#193] [Q&A] [answered]
All-time trouble for all simple things needs search all time. In these windows, I got the worker and his running and I check the log now is the window empty server running. Ho to set up to see a worker in this window?

![msedge_l7LX3qMTNC](https://user-images.githubusercontent.com/34956477/126021493-b1c7d149-7f21-4ab9-af1b-04d7b356b4ca.png)

How setup in plotman run 10 jobs like in madmax

-n 10 and I create 10 jobs in plotman i looking searching web page and cannot find 

I reinstall Machinaris for test version and worker back on regular machineries i reinstall stop-start docker copy backup  plotman.yaml nothing help and worker not shows reinstall to test version and is back instant no any changes
![msedge_gXJoH8zPAu](https://user-images.githubusercontent.com/34956477/126021781-a4bb6806-d4bc-4710-80ad-4b71d7f22dbf.png)

Where in plowman i got parameters about how many jobs I want to create but no parallel
![msedge_n3iQZmuLRc](https://user-images.githubusercontent.com/34956477/126021865-a060f066-ed5d-4ee2-a229-9c4cfe723c6d.png)
 one done start second

**Comment by guydavis** [comment, marked as answer]
Glad to hear you were able to get this sorted out via our Discord server.  Sorry for the trouble.  

An upcoming release of Machinaris will provide better error messages regarding Plotman configuration issues, to avoid the problems above.

Please update this if you still have issues here.  Else, please mark this Question as Answered.
